src/App.js

```diff
@@ -10,10 +14,14 @@ class App extends React.PureComponent {
 
   render() {
     return (
-      <div>
-        <h1>CALCULATOR</h1>
-        <Calculator />
-      </div>
+      <Router>
+        <NavigationBar />
+        <Routes>
+          <Route exact path="src/components/pages/Home.js" element={<Home />} />
+          <Route path="/components/Calculator" element={<Calculator />} />
+          <Route path="src/components/pages/quote.js" element={<Quote />} />
```

Comment:
- [ ] [optional] It would have ben better if you reduced the length of your path and change the home page route so that when the app is launched it doesn't show a blank screen with only navbar displayed:
```suggestion
          <Route exact path="/" element={<Home />} />
          <Route path="/calculator" element={<Calculator />} />
          <Route path="quote.js" element={<Quote />} />
```